src/items/FilePdf.tsx

```diff
@@ -41,11 +41,13 @@ const FilePdf: FC<FilePdfProps> = ({
     }
   };
 
+  const src = `https://pdfviewer.dev.graasp.org/pdf/web/viewer.html?file=${url}`;
```

Comment:
Do you perform checks on the receiving end to prevent injection? Also, should this always be served from the dev domain?